src/components/interpolatedColorLegend.ts

```diff
@@ -27,13 +27,13 @@ export module Components {
      * Creates an InterpolatedColorLegend.
```

Comment:
Huge aside - Should `InterpolatedColorLegend` actually be called `InterpolatedLegend` instead?